docs/message/message-schema.yaml

```diff
@@ -0,0 +1,70 @@
+definitions:
+  messages:
+    type: array
+    items:
+      type: object
+      properties:
+        _id:
+          type: string
+        author:
+          type: string
+          ref: '#/components/user'
+        authorRole:
+          type: string
+          enum:
+              - student
+              - tutor
+        messageText:
+          type: string
+        isRead:
+          type: boolean
+        isNotified:
+          type: boolean
+        chat:
+          type: string
+          ref: '#/components/chat'
+        createdAt:
+          type: string
+          format: date-time
+        updatedAt:
+          type: string
+          format: date-time
+  messageBody:
+    type: object
+    properties:
+      author:
+        type: string
+        ref: '#/components/user'      
+      messageText:
```

Comment:
Change filed name here to "text" as well

Reply:
Fixed